### ISSUE TITLE
Support docker plugins

### ISSUE THREAD

**Issue by genebean** [puppetlabs/puppetlabs-docker#139]
While reading https://vmware.github.io/vsphere-storage-for-docker/documentation/install.html I came across this chunk of code that I would like to automate via this module:

```bash
$ docker plugin install --grant-all-permissions --alias vsphere vmware/vsphere-storage-for-docker:latest

latest: Pulling from vmware/vsphere-storage-for-docker
f07d34084e57: Download complete
Digest: sha256:e1028b8570f37f374e8987d1a5b418e3c591e2cad155cc3b750e5e5ac643fb31
Status: Downloaded newer image for vmware/vsphere-storage-for-docker:latest
Installed plugin vmware/vsphere-storage-for-docker:latest
```

Since other installation and configuration of Docker is done via this module I figure this is a logical addition.

**Comment by scotty-c** [comment]
@genebean we will create a ticket in our backlog for this

**Comment by davejrt** [comment]
Fixed in PR #180
